src/mixins/filing-template-mixin.ts

```diff
@@ -225,7 +226,24 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
 
     // restore the amalgamating businesses array
     if (draftFiling.amalgamation.amalgamatingBusinesses) {
-      this.setAmalgamatingBusinesses(draftFiling.amalgamation.amalgamatingBusinesses)
+      this.setAmalgamatingBusinesses([
+        // *** TODO: remove static items when they are no longer needed for testing
```

Comment:
Leaving this in for now.

Reply:
Note: this "tech debt" and the other ones in comments above have been added to the ["DO THIS LAST" ticket](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/18689) for final resolution.